spectrometer.cabal

```diff
@@ -142,6 +142,9 @@ library
     App.Fossa.Container
     App.Fossa.Container.Analyze
     App.Fossa.Container.Test
+    App.Fossa.IAT.AssertUserDefinedBinaries
```

Comment:
[optional] [minor] Place`App.Fossa.IAT*` below `App.Fossa.FossaAPIV1` to preserve alphabetical order.